packages/fx-core/resource/package.nls.json

```diff
@@ -306,7 +308,7 @@
   "core.createProjectQuestion.projectType.copilotGroup.title": "Use Copilot",
   "core.createProjectQuestion.projectType.createGroup.title": "Create",
   "core.createProjectQuestion.projectType.declarativeCopilot.label": "Declarative Copilot",
-  "core.createProjectQuestion.projectType.declarativeCopilot.title": "Author a Declarative Copilot",
+  "core.createProjectQuestion.projectType.declarativeCopilot.detail": "Craft your own Copilot by declaring instructions, actions, and knowledge to customize Copilot for Microsoft 365.",
```

Comment:
Revised: Create your own copilot by declaring instructions, actions, & knowledge to suit your needs.